kubernetes-model/kubernetes-model/src/main/java/io/fabric8/kubernetes/api/model/Quantity.java

```diff
@@ -50,6 +50,7 @@
 @Buildable(editableEnabled = false, validationEnabled = false, generateBuilderPackage=true, builderPackage = "io.fabric8.kubernetes.api.builder", inline = @Inline(type = Doneable.class, prefix = "Doneable", value = "done"))
 public class Quantity  implements Serializable {
 
+  public static final String AT_LEAST_ONE_DIGIT_REGEX = ".*\\d+.*";
```

Comment:
This one should be private if possible
```suggestion
  private static final String AT_LEAST_ONE_DIGIT_REGEX = ".*\\d+.*";
```